.github/workflows/wheels.yml

```
@@ -71,6 +74,96 @@ jobs:
           name: dist
           path: ./wheelhouse/*.whl
 
+  windows:
+    name: Windows ${{ matrix.arch }}
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: x86
+            cibw_arch: x86
+          - arch: x64
+            cibw_arch: AMD64
+          - arch: ARM64
+            cibw_arch: ARM64
+    steps:
+      - uses: actions/checkout@v4
+      - name: Checkout extra test images
+        uses: actions/checkout@v4
+        with:
+          repository: python-pillow/test-images
+          path: Tests\test-images
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      - name: Prepare for build
+        run: |
+          choco install nasm --no-progress
+          echo "C:\Program Files\NASM" >> $env:GITHUB_PATH
+
+          # Install extra test images
+          xcopy /S /Y Tests\test-images\* Tests\images
+
+          & python.exe -m pip install cibuildwheel==2.16.2
+
+          # Cannot cross-compile FriBiDi (only used for tests)
+          $FLAGS = ("--no-imagequant", "--architecture=${{ matrix.arch }}")
+          if ('${{ matrix.arch }}' -eq 'ARM64') { $FLAGS += "--no-fribidi" }
+          & python.exe winbuild\build_prepare.py -v @FLAGS
+        shell: pwsh
+      - name: Build wheels
+        run: |
+          setlocal EnableDelayedExpansion
+          for %%f in (winbuild\build\license\*) do (
+            set x=%%~nf
+            rem Skip FriBiDi license, it is not included in the wheel.
+            set fribidi=!x:~0,7!
+            if NOT !fribidi!==fribidi (
+              rem Skip imagequant license, it is not included in the wheel.
+              set libimagequant=!x:~0,13!
+              if NOT !libimagequant!==libimagequant (
+                echo. >> LICENSE
+                echo ===== %%~nf ===== >> LICENSE
+                echo. >> LICENSE
+                type %%f >> LICENSE
+              )
+            )
+          )
+          call winbuild\\build\\build_env.cmd
+          %pythonLocation%\python.exe -m cibuildwheel . --output-dir wheelhouse
+        env:
+          CIBW_ARCHS: ${{ matrix.cibw_arch }}
+          CIBW_BEFORE_ALL: "{package}\\winbuild\\build\\build_dep_all.cmd"
+          CIBW_CACHE_PATH: "C:\\cibw"
+          CIBW_TEST_SKIP: "*-win_arm64"
+          CIBW_TEST_COMMAND: >-
+            docker run --rm
+            -v {project}:C:\pillow
+            -v C:\cibw:C:\cibw
+            -v %CD%\..\venv-test:%CD%\..\venv-test
+            -e CI -e GITHUB_ACTIONS
+            mcr.microsoft.com/windows/servercore:ltsc2022
+            powershell C:\pillow\.github\workflows\wheels-test.ps1 %CD%\..\venv-test
```

Still work with some extra indent for readability?
```suggestion
            docker run --rm
              -v {project}:C:\pillow
              -v C:\cibw:C:\cibw
              -v %CD%\..\venv-test:%CD%\..\venv-test
              -e CI -e GITHUB_ACTIONS
              mcr.microsoft.com/windows/servercore:ltsc2022
              powershell C:\pillow\.github\workflows\wheels-test.ps1 %CD%\..\venv-test
```

---

That adds extra newlines which [makes it fail](https://github.com/python-pillow/Pillow/actions/runs/7010925491/job/19072484245?pr=7580#step:6:3335).

Using a [single-quoted flow scalar](https://yaml-multiline.info#flow-scalars-single) seems to work fine:
```
          CIBW_TEST_COMMAND: 'docker run --rm
            -v {project}:C:\pillow
            -v C:\cibw:C:\cibw
            -v %CD%\..\venv-test:%CD%\..\venv-test
            -e CI -e GITHUB_ACTIONS
            mcr.microsoft.com/windows/servercore:ltsc2022
            powershell C:\pillow\.github\workflows\wheels-test.ps1 %CD%\..\venv-test'
```